manifests/02_csi_driver_operators_namespace.yaml

```diff
@@ -1,3 +1,4 @@
+# We keep this namespace without capability name untill we merge cluster-csi-snapshot-controller capability
```

Comment:
Namespace `openshift-cluster-csi-drivers` is not used by snapshot-controller, so this comment is wrong.

Question is, if CVO should create this namespace without `Storage` capability. It's used by optional CSI drivers, like AWS EFS or GCP Filestore, installed via OLM. IMO OLM should be able to create the namespace and therefore this namespace should have `Storage` capability.

@openshift/storage what do you think?

Reply:
If we go this way I would need to provide a namespace creation mechanism/rule to OLM, right?

Reply:
OLM already knows how to create a namespace.

Reply:
fixed